packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -54,22 +54,17 @@ const ButtonCore: React.ForwardRefExoticComponent<
             testId,
             type = undefined,
             spinner,
-            icon,
+            startIcon,
+            endIcon,
             id,
             waiting: _,
             ...restProps
         } = props;
 
-        const iconWidth = icon
-            ? size === "small"
-                ? theme.size.width.medium
-                : theme.size.width.large
-            : 0;
```

Comment:
I realized this isn't being used anywhere. It used to be used in `_generatestyles()`, but that got replaced by the `padding` value.

Reply:
oh nice catch!

Now that you got rid of this, I think you can also remove these tokens from the default theme: https://github.com/Khan/wonder-blocks/blob/1644f6510760c26c8ab407d49654faae36fddcc9/packages/wonder-blocks-button/src/themes/default.ts#L107-L110